### ISSUE TITLE
`pyrevit extensions update --all` does nothing

### ISSUE THREAD

**Issue by melnikovalex** [eirannejad/pyRevit#590]
**Describe the bug**
Hey Ehsan,

I've noticed that extensions in our office are not being updated properly and found such a strange thing. 

When I update **a certain** extention using CLI-Tool it works perfectly. But when I update with an option `--all` nothing happens and logs are almost empty.

**To Reproduce**
`pyrevit.exe extensions update --all` nothing is being updated, log:

```
2019-05-07 09:47:26.9873|DEBUG|pyRevitManager.Views.pyRevitCLI|Active Keyword: extensions
2019-05-07 09:47:27.0143|DEBUG|pyRevitManager.Views.pyRevitCLI|Active Keyword: update
```

`pyrevit.exe extensions update pyapex` works good, log:
```
2019-05-07 09:48:41.9376|DEBUG|pyRevitManager.Views.pyRevitCLI|Active Keyword: extensions
2019-05-07 09:48:41.9645|DEBUG|pyRevitManager.Views.pyRevitCLI|Active Keyword: update
2019-05-07 09:48:41.9875|DEBUG|pyRevitLabs.TargetApps.Revit.PyRevit|Looking up installed extension "pyapex"...
2019-05-07 09:48:41.9875|DEBUG|pyRevitLabs.TargetApps.Revit.PyRevit|Try getting config as list "core:userextensions"
2019-05-07 09:48:41.9875|DEBUG|pyRevitLabs.TargetApps.Revit.PyRevit|Try getting config "core:userextensions" ?? []
2019-05-07 09:48:42.0752|DEBUG|pyRevitLabs.TargetApps.Revit.PyRevit|Verified extension search path "C:\Users\aleksei.melnikov\AppData\Roaming\pyRevit\Extensions"
2019-05-07 09:48:42.1031|DEBUG|pyRevitLabs.TargetApps.Revit.PyRevit|Updating config "core:userextensions = ["C:\\Users\\aleksei.melnikov\\AppData\\Roaming\\pyRevit\\Extensions"]"
2019-05-07 09:48:42.1031|DEBUG|pyRevitLabs.TargetApps.Revit.PyRevit|Saving config file "C:\Users\aleksei.melnikov\AppData\Roaming\pyRevit\pyRevit_config.ini"
2019-05-07 09:48:42.1031|DEBUG|pyRevitLabs.TargetApps.Revit.PyRevit|Looking for installed extensions under "C:\Users\aleksei.melnikov\AppData\Roaming\pyRevit\Extensions"...
2019-05-07 09:48:42.1031|DEBUG|pyRevitLabs.TargetApps.Revit.PyRevit|Found installed extension "C:\Users\aleksei.melnikov\AppData\Roaming\pyRevit\Extensions\HWPy.extension"...
2019-05-07 09:48:42.1031|DEBUG|pyRevitLabs.TargetApps.Revit.PyRevit|Found installed extension "C:\Users\aleksei.melnikov\AppData\Roaming\pyRevit\Extensions\pyApex.extension"...
2019-05-07 09:48:42.1191|DEBUG|pyRevitLabs.TargetApps.Revit.PyRevit|"pyapex" Matched installed extension "pyApex"
2019-05-07 09:48:42.1191|DEBUG|pyRevitLabs.TargetApps.Revit.PyRevit|Updating extension "pyApex"
2019-05-07 09:48:42.1191|DEBUG|pyRevitLabs.TargetApps.Revit.PyRevit|Updating extension repo at "C:\Users\aleksei.melnikov\AppData\Roaming\pyRevit\Extensions\pyApex.extension"
2019-05-07 09:48:42.1191|DEBUG|pyRevitLabs.Common.GitInstaller|Force updating repo "C:\Users\aleksei.melnikov\AppData\Roaming\pyRevit\Extensions\pyApex.extension"...
2019-05-07 09:48:42.6498|DEBUG|pyRevitLabs.Common.GitInstaller|Repo "C:\Users\aleksei.melnikov\AppData\Roaming\pyRevit\Extensions\pyApex.extension" is already up to date.
```

**Desktop (please complete the following information):**
 - OS: Win 10 German
 - pyRevit Version 4.6.18
 - pyRevit Environment: 
```
==> Registered Clones (full git repos)
==> Registered Clones (deployed from archive)
==> Attachments
Error: pyRevit is attached to Revit 2018 but can not determine the clone and engine
Error: pyRevit is attached to Revit 2019 but can not determine the clone and engine
==> Installed UI Extensions
Name: "HWPy" | Repo: "" | Installed: "C:\Users\aleksei.melnikov\AppData\Roaming\pyRevit\Extensions\HWPy.extension"
Name: "pyApex" | Repo: "" | Installed: "C:\Users\aleksei.melnikov\AppData\Roaming\pyRevit\Extensions\pyApex.extension"
==> Installed Library Extensions
==> Default Extension Search Path
C:\Users\aleksei.melnikov\AppData\Roaming\pyRevit\Extensions
==> Extension Search Paths
C:\Users\aleksei.melnikov\AppData\Roaming\pyRevit\Extensions
==> Extension Sources - Default
https://github.com/eirannejad/pyRevit/raw/master/extensions/extensions.json
==> Extension Sources - Additional
X:\pyRevit_extensions.json
==> Installed Revits
Autodesk Revit 2018.3.1 | Version: 18.3.1.2 | Build: 20180423_1000(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2018"
Autodesk Revit 2019.2.1 | Version: 19.2.10.7 | Build: 20190225_1515(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2019"
==> Running Revit Instances
==> User Environment
Microsoft Windows 10 [Version 10.0.17763]
Executing User: HWPNET\aleksei.melnikov
Active User: HWPNET\aleksei.melnikov
Adming Access: No
%APPDATA%: "C:\Users\aleksei.melnikov\AppData\Roaming"
Latest Installed .Net Framework: "4.7.2"
Installed .Net Target Packs: v3.5 v4.0 v4.5 v4.5.1 v4.5.2 v4.6 v4.6.1 v4.6.2 v4.7.1 v4.X
pyRevit CLI 0.9.6.0
```

**Additional context**

I didn't manage to test it on a newer pyrevit version yet. I'll give an update if there are any changes.

Thank you!

**Comment by eirannejad** [comment]
Hmm. Okay I'll take a look

**Comment by eirannejad** [comment]
Just published 4.6.21. Please test and see if the changes has resolved this issue
https://github.com/eirannejad/pyRevit/releases/tag/v4.6.21

**Comment by eirannejad** [comment]
Resolved.
